packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -156,8 +168,8 @@ export class Integration {
    * Can be overridden by subclasses to allow the integration to interact with the method
    * being integrated, access the REST API object, method ARNs, etc.
    */
-  public bind(_method: Method) {
-    return;
+  public bind(_method: Method): IntegrationConfig {
```

Comment:
Why not `IntegrationConfig | undefined` for backwards compat and also returning empty objects is yakk.